fake_switches/cisco/command_processor/config_interface.py

```diff
@@ -128,6 +128,8 @@ def do_ip(self, *args):
             self.port.ip_redirect = True
         if "proxy-arp".startswith(args[0]):
             self.port.ip_proxy_arp = True
+        if "verify unicast".startswith(args[0]) and isinstance(self.port, VlanPort):
```

Comment:
I'm pretty sure that we split on space and that arg[0] will be "verify" and arg[1] will be "unicast".  So "verify multicast" for example will still fall in this if statement.